modular_citadel/code/game/machinery/wishgranter.dm

```diff
@@ -0,0 +1,52 @@
+/obj/machinery/wish_granter/attack_hand(mob/living/carbon/user)
+	if(charges <= 0)
+		to_chat(user, "The Wish Granter lies silent.")
+		return
+
+	else if(!ishuman(user))
+		to_chat(user, "You feel a dark stirring inside of the Wish Granter, something you want nothing of. Your instincts are better than any man's.")
+		return
+
+	else if(is_special_character(user))
+		to_chat(user, "Even to a heart as dark as yours, you know nothing good will come of this.  Something instinctual makes you pull away.")
+
+	else if (!insisting)
+		to_chat(user, "Your first touch makes the Wish Granter stir, listening to you.  Are you really sure you want to do this?")
+		insisting++
+
+	else
+		to_chat(user, "You speak.  [pick("I want the station to disappear","Humanity is corrupt, mankind must be destroyed","I want to be rich", "I want to rule the world","I want immortality.")].  The Wish Granter answers.")
+		if(alert("Are you really sure?", "Confirm decision", "Yes", "No") == "Yes")
+			if(alert("Are you really, really sure?", "Confirm decision", "Yes", "No") == "Yes")
+				if(alert("You have a chance of being dusted if you keep clicking yes. Are you sure?", "Confirm decision", "Yes", "No") == "Yes")
+					if(alert("Do you want to self-antag so badly you'd risk being dusted?", "Confirm decision", "Yes", "No") == "Yes")
+						if(alert("Really?", "Confirm decision", "Yes", "No") == "Yes")
+							if (prob(75))
+								to_chat(user, "Your head pounds for a moment, before your vision clears. You instantly regret all of your life choices up to this point. Your flesh rapidly flashes away into dust.")
```

Comment:
"Your head pounds for a moment before your vision clears. You instantly regret all of your life choices up to this point as you watch your flesh rapidly flash away into dust." Possible alteration.

Reply:
Even better.
"You fool. You knew this was bad from the beginning. Now you have nothing."